pymc/model_graph.py

```diff
@@ -357,26 +409,31 @@ def make_graph(
         node_formatters = update_node_formatters(node_formatters)
 
         graph = graphviz.Digraph(self.model.name)
-        for plate_label, all_var_names in self.get_plates(var_names).items():
-            if plate_label:
+        for plate_meta, all_vars in self.get_plates(var_names).items():
+            if plate_meta:
                 # must be preceded by 'cluster' to get a box around it
+                plate_label = create_plate_label(plate_meta, include_size=include_shape_size)
```

Comment:
Noticing that the plate_label actually depends on var_name in case of previous "{var_name}_dim{d}". However, the plate_label is required before the looping of `all_var_names`. i.e. `all_var_names` is assumed to be one element? Maybe that should be an explicit case?

Reply:
Didn't manage to follow, can you explain again?

Reply:
The `graph.subgraph` name of \"cluster\" + plate_label is dependent on the var_name which used to be constructed in the `get_plates` method (the previous keys).

However, after the subgraph is constructed, the `all_var_names` is looped over. This is assuming that `all_var_names` is only one element since the plate_label is used in the subgraph name.

Reply:
The `graph.subgraph` name of \"cluster\" + plate_label is dependent on the var_name which used to be constructed in the `get_plates` method (the previous keys).

However, after the subgraph is constructed, the `all_var_names` is looped over. This is assuming that `all_var_names` is only one element since the plate_label is used in the subgraph name.

Reply:
The `graph.subgraph` name of \"cluster\" + plate_label is dependent on the var_name which used to be constructed in the `get_plates` method (the previous keys).

However, after the subgraph is constructed, the `all_var_names` is looped over. This is assuming that `all_var_names` is only one element since the plate_label is used in the subgraph name.

Reply:
The `graph.subgraph` name of \"cluster\" + plate_label is dependent on the var_name which used to be constructed in the `get_plates` method (the previous keys of dictionary where the plate_label).

However, after the subgraph is constructed, the `all_var_names` is looped over. This is assuming that `all_var_names` is only one element since the plate_label is used in the subgraph name.